var/spack/repos/builtin/packages/mumps/package.py

```diff
@@ -125,28 +128,49 @@ def write_makefile_inc(self):
         fpic = '-fPIC' if '+shared' in self.spec else ''
         # TODO: test this part, it needs a full blas, scalapack and
         # partitionning environment with 64bit integers
-        if '+int64' in self.spec:
-            makefile_conf.extend(
-                # the fortran compilation flags most probably are
-                # working only for intel and gnu compilers this is
-                # perhaps something the compiler should provide
-                ['OPTF    = %s -O  -DALLOW_NON_INIT %s' % (fpic, '-fdefault-integer-8' if self.compiler.name == "gcc" else '-i8'),  # noqa
-                 'OPTL    = %s -O ' % fpic,
-                 'OPTC    = %s -O -DINTSIZE64' % fpic])
+
+        if '+idx64' in self.spec:
```

Comment:
It was a mistake while I was migrating from my old repository. It is now updated.